pandas/core/arrays/arrow/array.py

```diff
@@ -829,13 +829,9 @@ def _reduce(self, name: str, *, skipna: bool = True, **kwargs):
         """
         if name == "sem":
 
-            def pyarrow_meth(data, skipna, **kwargs):
-                numerator = pc.stddev(data, skip_nulls=skipna, **kwargs)
-                denominator = pc.sqrt_checked(
-                    pc.subtract_checked(
-                        pc.count(self._data, skip_nulls=skipna), kwargs["ddof"]
```

Comment:
is `kwargs['ddof']` not relevant here?

Reply:
Yeah I suppose this only applies in the `stddev` call.

When comparing to `ser.astype("Float64").sem()` the results were off, and removing `kwargs["ddof"]` aligned this pyarrow sem to nullable dtype sem.

Reply:
the test changes all look good.  this change i dont know enough to form an opinion on.  if youre confident its right then LGTM.  otherwise lets find an appropriate person to double-check

Reply:
I think based on the `nanops.nansem` implementation, I'm fairly sure these align

https://github.com/pandas-dev/pandas/blob/3fffb6d49abe20ebc4a3380181f90103fb9ce22e/pandas/core/nanops.py#L1033-L1042